anomalib/models/dfm/lightning_model.py

```diff
@@ -37,6 +38,7 @@ class Dfm(AnomalyModule):
     def __init__(
         self,
         feature_extractor: FeatureExtractorParams,
+        input_size: Tuple[int, int],
```

Comment:
Instead of repeating these changes here, would it make sense to rebase the feature branch with main? That would save us some merge conflicts in the future

Reply:
I'll do that